mangaki/mangaki/models.py

```diff
@@ -10,6 +10,179 @@
 from mangaki.discourse import get_discourse_data
 from mangaki.choices import ORIGIN_CHOICES, TYPE_CHOICES, TOP_CATEGORY_CHOICES
 from mangaki.utils.ranking import TOP_MIN_RATINGS, RANDOM_MIN_RATINGS, RANDOM_MAX_DISLIKES, RANDOM_RATIO
+#from mangaki.utils.dpp import *
+
+from sklearn.utils.extmath import randomized_svd
+from scipy.spatial.distance import pdist, squareform
+from numpy.random import choice
+
+import numpy as np
+from scipy.sparse import csc_matrix
+from mangaki.utils.values import rating_values
+
+import pandas
+
+
+class BuildMatrix(object):
```

Comment:
No need to inherit from `object` in python3. Can we have a better name ? Maybe `RatingsMatrix` ?